vta/python/vta/top/graphpack.py

```diff
@@ -309,18 +329,30 @@ def graph_pack(expr,
         The bit-width of the weights.
 
     start_name: str, optional
-       Start packing from certain known node.
+       Start packing from certain known node when start_name_idx is None.
 
     stop_name: str, optional
-       Stop packing from certain known node.
+       Stop packing from certain known node when stop_name_idx is None.
+
+    start_name_idx: str, optional
+        When start_name_idx not None, start packing only when node name equal start_name
+        and node idx equal start_name_idx.
```

Comment:
node index equals to start_name_idx

Reply:
fixed.

Reply:
I meant exactly "node index equals to start_name_idx", or am I wrong in the suggested English express?